packages/react-sdk/src/components/ImageUpload/useSlideImageUpload.test.tsx

```diff
@@ -90,14 +100,14 @@ describe('useSlideImageUpload', () => {
   });
 
   it('should not explode if there is an upload error', async () => {
-    jest.mocked(console.error).mockImplementation(() => {});
+    consoleSpy.mockImplementation(() => {});
     render(<TestFileInput />, { wrapper: Wrapper });
 
     const file = new File([], 'example.jpg', {
       type: 'image/jpeg',
     });
     const uploadError = new Error('upload error');
-    jest.mocked(file.arrayBuffer).mockImplementation(() => {
+    vi.spyOn(file, 'arrayBuffer').mockImplementation(() => {
```

Comment:
Mocked because of https://github.com/nordeck/matrix-neoboard/blob/819e85d26695fe7f9b43db61a61bd05644b2da6b/packages/react-sdk/src/setupTests.ts#L106

```suggestion
    vi.mocked(file, 'arrayBuffer').mockImplementation(() => {
```